modules/position/src/main/native/android/dalvik/DalvikPositionService.java

```diff
@@ -79,7 +79,7 @@ public DalvikPositionService(Activity activity) {
         if (debug) {
             Log.v(TAG, "Construct DalvikPositionService");
         }
-        boolean gpsEnabled = Util.verifyPermissions(Manifest.permission.ACCESS_COARSE_LOCATION) || 
+        boolean gpsEnabled = Util.verifyPermissions(Manifest.permission.ACCESS_COARSE_LOCATION) | // Both permissions require to be verified
```

Comment:
On a second thought, this might be better (to avoid future confusions):
```
// Verify both permissions
boolean coarseEnabled = Util.verifyPermissions(Manifest.permission.ACCESS_COARSE_LOCATION);
boolean fineEnabled = Util.verifyPermissions(Manifest.permission.ACCESS_FINE_LOCATION);
boolean gpsEnabled = coarseEnabled || fineEnabled;
```
Would you mind changing it?

Reply:
No, I actually had the same idea but just did not want to change too much. Just pushed the changes.